.github/workflows/cpu-ci.yml

```diff
@@ -33,6 +33,7 @@ jobs:
         python -m pip install --upgrade pip setuptools==59.4.0 wheel 
         python -m pip install -r requirements-dev.txt
         python -m pip install tensorflow
+        python -m pip install xgboost
```

Comment:
Now that we're supporting more than one modeling library, we should make `tensorflow` and `xgboost` optional dependencies. For this PR, probably good enough to make `xgboost` optional and we do the same for TF in a separate PR.

Reply:
can you clarify what you mean by optional in the context of this line. The systems package currently only has two dependencies, merlin-core and nvtabular. Do you mean adding something like we have with models where we can pip install merlin-models[xgboost] ? 

the FIL op as currently written doesn't strictly depend on xgboost (import-wise), only in the sense that some calling it will need to load a model - maybe that could be left up to them to install?

Reply:
Yeah, I think we might want to set up `pip install merlin-systems[xgboost]`, but it's out of scope for this PR. This line just reminded me that it's a thing to do.